plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py

```diff
@@ -84,26 +84,24 @@ class RsyncConf:
 
 
 @dataclass
-class PluginMandatoryInstallConf:
-    hydra_version: str = hydra.__version__
-    ray_version: str = ray.__version__
-    cloudpickle_version: str = cloudpickle.__version__
-    omegaconf_version: str = omegaconf.__version__
-    pickle5_version: str = pkg_resources.get_distribution("pickle5").version
-    hydra_ray_launcher_version: str = pkg_resources.get_distribution(
-        "hydra_ray_launcher"
-    ).version
+class MandatoryInstallConf:
```

Comment:
This class looks like EnvSetupConf more than it looks like MandatoryInstallConf :)